ci/pipeline.yml

```diff
@@ -4,7 +4,7 @@ resources:
   type: git
   source:
     uri: https://github.com/cloud-gov/cg-sandbox-bot
-    branch: main
+    branch: gem-hunt #main
```

Comment:
Need to change back to main, since I assume you're done making changes

Reply:
I will, there are two pending questions left:

 - Running both create users together (I have a reason, want to make sure you are ok with it)
 - Running this in prod - I'm ok with it as long as there is some sort of consensus and we are fine with using fedramp (or someone else to be selected) as the test guinea pig

Reply:
Yeah, I commented on those, but:

- Your reason made sense
- Not going to block this PR on running in prod. We can discuss in engineering huddle

Reply:
Roger roger, 1 more commit inbound to flip branch